src/libraries/Microsoft.PowerFx.Interpreter/Functions/IAsyncTexlFunction2.cs

```diff
@@ -1,18 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT license.
-
-using System;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.PowerFx.Functions;
-using Microsoft.PowerFx.Types;
-
-namespace Microsoft.PowerFx.Core.Functions
-{
-    // A Texl function capable of async invokes. 
-    [Obsolete("This interface is obsolete and will be removed in a future release. Please use IAsyncConnectorTexlFunction instead.")]
-    internal interface IAsyncTexlFunction2
```

Comment:
>IAsyncTexlFunction2 [](http://example.com/codeflow?start=23&length=19)

to be safe - can we do in separate PR?

Reply:
I'll resolve this comment as it's already present in the PR